### PR TITLE
Updating the readme.md to make it easier to follow

### PR DESCRIPTION
**Pull request by MufcVarun11**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,11 +39,21 @@ Running on a dedicated device in a car
...
+  To install the release version of Openpilot, use the URL (https://github.com/commaai/openpilot).
+  To install Openpilot master for more advanced users, use the URL (https://installer.comma.ai/commaai/master). You can replace "commaai" with another GitHub username to install a fork.
+
+3. Supported Car:
+  Ensure that you have one of [the 250+ supported cars](docs/CARS.md). Openpilot supports a wide range of car makes including Honda, Toyota, Hyundai, Nissan, Kia, Chrysler, Lexus, Acura, Audi, VW, Ford, and many more.
+  If your car is not officially listed as supported but has adaptive cruise control and lane-keeping assist, it's likely capable of running Openpilot.
+  
+4. Car Harness:
+  You will also need a [car harness](https://comma.ai/shop/car-harness) to connect your Comma 3X or Comma Three to your car.
+  Make sure to follow the manufacturer's instructions and safety guidelines when installing and using Openpilot with your supported device and vehicle.
```
Comment (haraschax) [inline README.md:56]: This isn't needed.

File: README.md [hunk 2]
```diff
@@ -39,11 +39,21 @@ Running on a dedicated device in a car
...
-The URL, openpilot.comma.ai will install the release version of openpilot. To install openpilot master, you can use installer.comma.ai/commaai/master, and replacing commaai with another GitHub username can install a fork.
-* One of [the 250+ supported cars](docs/CARS.md). We support Honda, Toyota, Hyundai, Nissan, Kia, Chrysler, Lexus, Acura, Audi, VW, Ford and more. If your car is not supported but has adaptive cruise control and lane-keeping assist, it's likely able to run openpilot.
-* A [car harness](https://comma.ai/shop/products/car-harness) to connect to your car.
+1. Supported Device: 
+  A Comma 3X or Comma Three. You can purchase these devices from (https://comma.ai/shop/comma-3x)
+  
+2. Software Installation:
+  The setup procedure for the Comma 3/3X allows users to enter a URL for custom software installation.
+  To install the release version of Openpilot, use the URL (https://github.com/commaai/openpilot).
+  To install Openpilot master for more advanced users, use the URL (https://installer.comma.ai/commaai/master). You can replace "commaai" with another GitHub username to install a fork.
```
Comment (haraschax) [inline README.md:48]: openpilot is stylized without a capital O

File: README.md (hunk 1 above)
Comment (MufcVarun11) [reply README.md]: Thank you. Done with the changes :)

File: README.md [hunk 3]
```diff
@@ -39,13 +39,21 @@ Running on a dedicated device in a car
...
-* One of [the 250+ supported cars](docs/CARS.md). We support Honda, Toyota, Hyundai, Nissan, Kia, Chrysler, Lexus, Acura, Audi, VW, Ford and more. If your car is not supported but has adaptive cruise control and lane-keeping assist, it's likely able to run openpilot.
-* A [car harness](https://comma.ai/shop/products/car-harness) to connect to your car.
-
-We have detailed instructions for [how to mount the device in a car](https://comma.ai/setup).
+1. Supported Device: 
+  A Comma 3X or Comma Three. You can purchase these devices from (https://comma.ai/shop/comma-3x)
+  
+2. Software Installation:
+  The setup procedure for the Comma 3/3X allows users to enter a URL for custom software installation.
+  To install the release version of openpilot, use the URL (https://github.com/commaai/openpilot).
```
Comment (incognitojam) [inline README.md:47]:
```suggestion
  To install the release version of openpilot, use the URL `openpilot.comma.ai`.

File: README.md [hunk 4]
```diff
@@ -39,13 +39,21 @@ Running on a dedicated device in a car
...
+  To install openpilot master for more advanced users, use the URL (https://installer.comma.ai/commaai/master). You can replace "commaai" with another GitHub username to install a fork.
```
Comment (incognitojam) [inline README.md:48]:
```suggestion
  To install openpilot master for more advanced users, use the URL `installer.comma.ai/commaai/master`. You can replace "commaai" with another GitHub username to install a fork.
```

File: README.md [hunk 5]
```diff
@@ -39,13 +39,21 @@ Running on a dedicated device in a car
...
 To use openpilot in a car, you need four things
-* A supported device to run this software: a [comma 3X](https://comma.ai/shop/comma-3x) or comma three.
-* This software. The setup procedure of the comma 3/3X allows the user to enter a URL for custom software.
-The URL, openpilot.comma.ai will install the release version of openpilot. To install openpilot master, you can use installer.comma.ai/commaai/master, and replacing commaai with another GitHub username can install a fork.
-* One of [the 250+ supported cars](docs/CARS.md). We support Honda, Toyota, Hyundai, Nissan, Kia, Chrysler, Lexus, Acura, Audi, VW, Ford and more. If your car is not supported but has adaptive cruise control and lane-keeping assist, it's likely able to run openpilot.
-* A [car harness](https://comma.ai/shop/products/car-harness) to connect to your car.
-
-We have detailed instructions for [how to mount the device in a car](https://comma.ai/setup).
+1. Supported Device: 
+  A Comma 3X or Comma Three. You can purchase these devices from (https://comma.ai/shop/comma-3x)
```
Comment (incognitojam) [inline README.md:43]: Can you change the comma 3/3X back? comma is lowercase too

**Pull request merged**